tests/test_usb_iss.py

```diff
@@ -48,7 +48,7 @@ def test_setup_i2c(self):
                 io2_type=defs.IOType.OUTPUT_HIGH)
 
             assert_that(self.driver.write_cmd,
-                        called_with(0x5A, [0x02, i2c_mode, 0x04]))
+                        called_with(0x5A, [0x02, i2c_mode, 0xA4]))

```

Comment:
Note that the code now sets a couple of extra bits in the `IO_TYPE` field. I don't think this matters, but wondered if you've checked this on real hardware?

Reply:
I haven't had the chance to check with actual hardware yet, although to be safe we should probably mask these bits to be consistent with the datasheet. I'll add another commit to fix this (and the None stuff too while I'm at it).